packages/gatsby-plugin-utils/src/polyfill-remote-file/utils/url-generator.ts

```diff
@@ -46,6 +48,13 @@ export function generateImageArgs({
       `crop=${Array.isArray(cropFocus) ? cropFocus.join(`,`) : cropFocus}`
     )
   }
+  if (aspectRatio) {
+    // crop must be set for aspectRatio to work
+    if (!cropFocus) {
+      args.push(`fit=crop`)
```

Comment:
There is already a `fit` parameter in `gatsbyImageResolver` that should be used instead of the hard coded fit=crop. 

Also -  fit should be set even if there isn't an aspect ratio

Reply:
This is actually a bit of a special case. We use `fit` only to calculate `width` and height` so that locally processed images and those processed with Image CDN on Gatsby Cloud are identical in terms of size and crop.

Because of the way that `aspectRatio` is handled on Gatsby Cloud, it is required that fit be either "crop" or "fill". Here I make sure that the `fit` param is added if it does not exist already so that `aspectRatio` will actually be used on the Gatsby Cloud side